channels/in.m3u

```diff
@@ -411,6 +417,10 @@ http://cdn.clive.in:1935/sadhnabhakti/sadhnabhakti.stream_HDp/playlist.m3u8
 http://cdn.clive.in:1935/sadhnaplus/sadhnaplus.stream_HDp/media.m3u8
 #EXTINF:-1 tvg-id="SafariTV.in" tvg-name="Safari TV" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/LOepeP3.jpg" group-title="",Safari TV (480p)
 https://j78dp346yq5r-hls-live.5centscdn.com/safari/live.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="SanaTV.in" tvg-name="Sana TV" tvg-country="IN" tvg-language="Tamil" tvg-logo="" group-title="Entertainment",Sana TV
+http://media.7starcloud.com:1935/live/sanatv2/chunklist_w1266678806.m3u8
+#EXTINF:-1 tvg-id="SanaPlus.in" tvg-name="Sana Plus" tvg-country="IN" tvg-language="Tamil" tvg-logo="" group-title="Music",Sana Plus
+http://media.7starcloud.com:1935/live/sanatv2/chunklist_w1266678806.m3u8
```

Comment:
Umm... I see the same links right there.

Reply:
http://hdserver.7starcloud.com:1935/sanatv/sanatv/chunklist_w1434725736.m3u8

Reply:
@LaneSh4d0w Here is the link for Sana Tv,
http://hdserver.7starcloud.com:1935/sanatv/sanatv/chunklist_w1434725736.m3u8

Reply:
Changed.